docs/usage-limits.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: usage-limits
+append_help_link: true
+hide_title: true
+description: >-
+  Frequently asked questions about Semgrep, comparisons to similar tools,
+  rule licensing, technical support, and more.
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Usage limits
+
+This section describes usage limits.
+
+:::caution important dates
+* The Semgrep Community tier will be replaced with Team tier on May 23, 2023.
+:::
+
+### What is changing about Semgrep tiers?
+
+We are sunsetting the Community tier and offering our Team tier for free to all users, subject to a usage limit of 10 contributors scanning on private repositories. A contributor is someone who has contributed code to a private repository being scanned by Semgrep.
```

Comment:
@s-santillan Do not use "being scanned" but "scanned". The same applies to "being analyzed" or "being written" - you can use "analyzed" or "written". Apply this to all similar grammatical structures.

Also, avoid using "we" when possible.